Extension/Configuration/JsonFileService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using ModLib.Debugging;
+using Newtonsoft.Json;
+using TaleWorlds.Engine;
+using YAPO.Configuration.Models;
+using Path = System.IO.Path;
+
+namespace YAPO.Configuration
+{
+    public static class JsonFileService
```

Comment:
Since this only provides save and load methods for the sorter configuration it isn't really a general JsonFileService as the name implies